lms/static/sass/elements/_controls.scss

```diff
@@ -323,3 +323,118 @@
       margin-bottom: none;
     }
   }
+
+
+// ===================
+// Dropdown menus
+// ===================
+.wrapper-more-actions {
+    @include float(right);
+    @include margin-right(($baseline/2));
+    position: relative;
+    display: inline-block;
+
+    .button-more.has-dropdown {
+        @extend %ui-depth2;
+        @include transition(all, $tmg-f2);
+        position: relative;
+        padding: ($baseline/2);
+        border: 1px solid transparent;
+        background: $white;
+        box-shadow: none;
+        font-size: $body-font-size;
+        text-shadow: none; // Clears text shadows on all buttons
+        line-height: ($baseline/2);
+
+        .icon {
+            @include transition(all, $tmg-f2);
+            @include transform-origin(45% center); // odd first value to keep icons centered
```

Comment:
Hmm, I wonder if the ``line-height`` is causing you to have to do this (rather than using ``@include transform-origin(center center)``. Mind trying to set a ``line-height: 0;`` on the icon and seeing if that plays better with the ``center center`` approach?